src/components/Aqidata.jsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import millify from 'millify';
+import Header from './Header';
+import { displayWeatherData } from '../redux/Aqidata/aqidataSlice';
+
+const Aqiweather = () => {
+  const location = useLocation();
+  const { city } = location.state;
+  const { weather } = useSelector((state) => state.weather);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const obj = {
+      lat: city.coordinates[0],
+      lon: city.coordinates[1],
+    };
+    dispatch(displayWeatherData(obj));
+  }, [city.coordinates, dispatch]);
+
+  const weatherList = weather?.list || [];
+
+  return (
+    <div className="bg-[#e94988]">
+      <Header title={city.cityName[0]} />
+      <table className="table-fixed bg-[#e94988] w-full mb-4">
+        <thead>
+          <tr className="bg-[#e94988] text-white">
+            <td>
+              <div className="flex justify-between items-start">
+                <div className="px-3 py-4 items-start flex cursor-pointer">
+                  <img src={city.flag} alt={city.alt} width="200" />
+                </div>
+                <div className="px-3 py-4 flex flex-col items-end cursor-pointer">
+                  <h2 className="font-semibold">Country</h2>
+                  <p>{city.country}</p>
+                  <h2 className="font-semibold">Population</h2>
+                  <p>{millify(city.population)}</p>
+                </div>
+              </div>
+            </td>
+          </tr>
+          <tr>
+            <th>
+              <div className="flex justify-center items-center">
+                <div className="px-3 py-4 items-center flex cursor-pointer">
+                  <h4 className="text-white">Weather Data</h4>
+                </div>
+              </div>
+            </th>
+          </tr>
+        </thead>
+        {weatherList.map((weatherData) => (
+          <tbody key={weatherData.dt}>
+            <tr className="bg-[#df4782] text-white">
+              <td className="w-1/6 whitespace-wrap px-2 py-2">
+                <div className="flex items-start justify-between">
+                  <h2 className="font-bold">AQI</h2>
+                  <h3>{weatherData.main.aqi}</h3>
+                </div>
+              </td>
+            </tr>
+            <tr className="bg-[#e94988] text-white">
+              <td className="w-1/6 whitespace-wrap px-2 py-2">
+                <div className="flex items-start justify-between">
+                  <h2 className="font-bold">CO</h2>
+                  <h3>{weatherData.components.co}</h3>
+                </div>
+              </td>
+            </tr>
```

Comment:
- [ ] Great job so far implementing all the changes suggested by the previous reviewer. However, there are still changes you need to make in your design to improve it and make it match that of the given design in the project requirements. To do so kindly consider the below suggestions:

    - [ ] Upon comparing your design with that of the given one I noticed from the homepage that you're missing an `image`. On the given design each card on the homepage contains `images`, `text`, and `icon` but on your design, your homepage contains `text` and `icon` with no `image`. Kindly consider adding an image on each card on your homepage. See below for more details:
 
 <table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="222" alt="Screenshot 2023-08-25 at 08 46 05" src="https://github.com/Kanyorok/air-pollution-aqi/assets/107192076/d336cf8b-6987-4cb2-9e6b-841169dcc5d8">
</td>
<td>
<img width="339" alt="Screenshot 2023-08-25 at 08 49 16" src="https://github.com/Kanyorok/air-pollution-aqi/assets/107192076/e4f37899-dbd6-4085-860d-e621187e5cbf">
</td>
</tr>
</table>